requirements.txt

```diff
@@ -3,7 +3,7 @@ Flask==0.10.1
 itsdangerous==0.24
 Jinja2==2.8
 MarkupSafe==0.23
-MySQL-python==1.2.5
+mysqlclient==1.3.12
```

Comment:
We would need to install this on the production environment right?

Reply:
Yep. Good point.